pkg/services/searchV2/index.go

```diff
@@ -256,7 +269,7 @@ func (i *searchIndex) run(ctx context.Context, orgIDs []int64, reIndexSignalCh c
 
 				started := time.Now()
 				i.logger.Info("Start re-indexing")
-				i.reIndexFromScratch(ctx)
+				i.reIndexFromScratch(fullReindexCtx)
 				i.logger.Info("Full re-indexing finished", "fullReIndexElapsed", time.Since(started))
```

Comment:
Could you please extract traceID and log it here as well? 
Similar to https://github.com/grafana/grafana/blob/main/pkg/middleware/logger.go#L77

If we log the traceID we can quickly navigate from log lines to the trace view :) super powerful!

Reply:
done!